src/selection.ts

```diff
@@ -1,35 +1,55 @@
 import {SingleDefChannel} from './channel';
 import {VgBinding} from './vega.schema';
 
-
 export type SelectionTypes = 'single' | 'multi' | 'interval';
 export type SelectionResolutions = 'global' | 'independent' | 'union' |
   'union_others' | 'intersect' | 'intersect_others';
 
 export interface BaseSelectionDef {
-  // domain?: SelectionDomain;
-  resolve?: SelectionResolutions;
   on?: any;
+  resolve?: SelectionResolutions;
   // predicate?: string;
-  bind?: 'scales' | VgBinding | {[key: string]: VgBinding};
+  // domain?: SelectionDomain;
 
   // Transforms
   fields?: string[];
   encodings?: SingleDefChannel[];
+}
+
+export interface BaseSingleSelectionDef extends BaseSelectionDef {
```

Comment:
Should we name this `SingleSelectionConfig` so it shows up in the schema as `SingleSelectionConfig`?  (Same for multi/interval)